web/src/gql/queries/user.ts

```diff
@@ -0,0 +1,116 @@
+import { gql } from "@apollo/client";
+
+export const SEARCH_USER = gql`
+  query searchUser($nameOrEmail: String!) {
+    searchUser(nameOrEmail: $nameOrEmail) {
```

Comment:
Keeping it same as Re:Earth
```suggestion
export const GET_USER_BY_SEARCH = gql`
  query GetUserBySearch($nameOrEmail: String!) {
    searchUser(nameOrEmail: $nameOrEmail) {
```